common/it/com/commercetools/sunrise/common/version/UserAgentIntegrationTest.java

```diff
@@ -0,0 +1,56 @@
+package com.commercetools.sunrise.common.version;
```

Comment:
I've missed that. This package is only for the controller that provides the version of the app via an endpoint. The solution info sent via user-agent is not related with that controller, so I would place the classes somewhere else, for example in `com.commercetools.sunrise.ctp.useragent`